cli/tests/conftest.py

```diff
@@ -331,8 +331,8 @@ def as_snapshot(self, mask: Optional[Maskers] = None):
             sections["stdout - plain"] == sections["stdout - color"]
             and sections["stderr - plain"] == sections["stderr - color"]
         ):
-            del sections["stdout - color"]
-            del sections["stderr - color"]
+            sections["stdout - color"] = "<same as above: stdout - plain>"
```

Comment:
Would have been nice to also put that in a separate PR because this PR is already big.

Reply:
> Would have been nice to also put that in a separate PR because this PR is already big.

PRs are too heavy. If CI checks took less than 5 min, then yes, I'd make another PR. Right now, it's this or doing nothing at all, with people generally erring toward the latter.